### PR TITLE
Master ryanmarchand

### PR DESCRIPTION
**Pull request by rmarchandTen24**
This change is [Reviewable](https://reviewable.io/reviews/thirdchannel/code-challenge/7)

### CODE REVIEW

File: parser.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module.exports.templateParser=(templateString,templateData)=>{
+    var customTagsRegex = /\(\((#[a-z]+ )?[a-z]+.[a-z]*\)\)/g;
```
Comment (eliottwiener-thirdchannel) [inline parser.js:2]: Could you explain this regex?

Comment (rmarchandTen24) [reply parser.js:2]: https://regexr.com/4gnhc

Comment (eliottwiener-thirdchannel) [reply parser.js:2]: What is the purpose of the first capture group, `(#[a-z]+ )`?

Can you make this match a greater variety of placeholders such as `((x))`, `((TEST))`, `((manufacturer-product-code))`, `((2nd-name))`?

Comment (rmarchandTen24) [reply parser.js:2]: yeah i guess the first capture group doesn't add value
```suggestion
    var customTagsRegex = /\(\(([a-zA-Z0-9-]+ )?.[a-zA-Z0-9-]*\)\)/g;
```
I updated the regexr link

**Pull request closed without merging**